styles/styles.css

```diff
@@ -460,19 +422,6 @@ main .section.align .default-content-wrapper h4 {
     padding-top: 30px;
   }
 
-  main .section.contact .default-content-wrapper {
-    padding: 20px 0 40px;
-  }
-
-  main .section.contact .default-content-wrapper p {
-    margin-bottom: 0;
-  }
-
-  main .section.contact .default-content-wrapper em {
-    font-size: 1.2em;
-    padding-bottom: 0;
-  }
```

Comment:
These changes are to style 'contact' section on /digitech page which are still needed.